docs/detectors/nuget.md

```diff
@@ -0,0 +1,15 @@
+# NuGet Detection
+
+## Requirements
+
+NuGet Detection depends on the following to successfully run: 
+
+- One or more `*.nuspec`, `*.nupkg`, `*.packages.config`, or `.*csproj` files.
```

Comment:
I'd break this down further:
- [`Nuget` detector looks for `*.nupkg`, `*.nuspec`, `nuget.config`, `paket.lock`][1]
- [`NuGetPackagesConfig` detector looks for `packages.config`][2]
- [`NuGetProjectCentric` detector looks for `project.assets.json`][3]

[1]: https://github.com/microsoft/component-detection/blob/13f3e9f32c94bf6189fbd0bfbdf2e68cc60fccd9/src/Microsoft.ComponentDetection.Detectors/nuget/NuGetComponentDetector.cs#L40
[2]: https://github.com/microsoft/component-detection/blob/13f3e9f32c94bf6189fbd0bfbdf2e68cc60fccd9/src/Microsoft.ComponentDetection.Detectors/nuget/NuGetPackagesConfigDetector.cs#L25
[3]: https://github.com/microsoft/component-detection/blob/13f3e9f32c94bf6189fbd0bfbdf2e68cc60fccd9/src/Microsoft.ComponentDetection.Detectors/nuget/NuGetProjectModelProjectCentricComponentDetector.cs#L205